SimTracker/SiStripDigitizer/python/customizeSiStripAPVDynamicGainSimulation.py

```diff
@@ -0,0 +1,23 @@
+import FWCore.ParameterSet.Config as cms
+
+#
+# activate the SiStrip AVP dynamic gain simulation and loads from DB the corresponding conditions
+#
+def activateSiStripAPVDynamicGain(process):
+
+    if hasattr(process,'mix') and hasattr(process.mix,'digitizers') and hasattr(process.mix.digitizers,'strip'):
+        print("activating SiStrip APV Dynamic Gain simulation")
+        process.mix.digitizers.strip.includeAPVSimulation = cms.bool(True)
+    if hasattr(process, "mixData") and hasattr(process.mixData, "workers") and hasattr(process.mixData.workers, "strip"):
+        print("activating SiStrip APV Dynamic Gain simulation (premixing)")
+        process.mixData.workers.strip.includeAPVSimulation = cms.bool(True)
```

Comment:
The `includeAPVSimulation` parameter exists already for `mix` and `mixData`, right? In that case dropping `cms.bool` would allow catching typos.

Reply:
done.